docs/demo/gaiden-css/variables/colors/demo.html

```diff
@@ -2,8 +2,9 @@
 <html lang="pt-br">
 <head>
   <meta charset="utf-8">
-  <link rel="stylesheet" href="/demo/css/base.css">
-  <link rel="stylesheet" href="/demo/css/colors.css">
+  <link rel="stylesheet" href="./../../../../../gaiden-css/ninja-gaiden.css">
+  <link rel="stylesheet" href="./../../../../../gaiden-css/base.css">
+  <link rel="stylesheet" href="./../../../../../gaiden-css/colors.css">
```

Comment:
@eduardojmatos just to know, why you use this approach here instead of `/gaiden-css/base.css` 
https://github.com/getninjas/gaiden/pull/13/files#diff-eef4ad3abee6f257ee7c5f1ca242eec5R6